runtime/src/bank.rs

```diff
@@ -1775,12 +1803,34 @@ impl Bank {
         let mut partitions = vec![];
         if parent_epoch < current_epoch {
             if current_slot_index > 0 {
+                // generate and push gapped partitions because some slots are skipped
                 let parent_last_slot_index = self.get_slots_in_epoch(parent_epoch) - 1;
+
+                // ... for parent epoch
                 partitions.push(self.partition_from_slot_indexes(
                     parent_slot_index,
                     parent_last_slot_index,
                     parent_epoch,
+                    true,
                 ));
+
+                // needed for test_rent_eager_across_epoch_with_gap_under_multi_epoch_cycle,
+                // which depends on OperatingMode::Stable
+                #[cfg(test)]
+                let should_enable = true;
+
+                #[cfg(not(test))]
+                let should_enable = match self.operating_mode() {
+                    OperatingMode::Development => true,
+                    OperatingMode::Preview => current_epoch >= 55,
```

Comment:
This is just tentative...

Reply:
Please just use Epoch::max_value() here as well.  Once this PR lands and is backported, then create two new issues for enabling the feature on testnet and mainnet-beta and assign them to me.   I'll then update `Epoch::max_value()` to a value that works as the specific cluster deployment comes closer.

Reply:
Thanks for the info! I think I've done this time really: https://github.com/solana-labs/solana/pull/10206/files#r437485896